tests/benchmarking/test_benchmarking.py

```diff
@@ -217,7 +338,7 @@ def test_deep_layers_large_graph(large_graph, benchmark):
 
 def test_find_descendants(large_graph, benchmark):
     result = _run_benchmark(benchmark, large_graph.find_descendants, "mypackage")
-    assert len(result) == 17348
+    assert len(result) == 28222
```

Comment:
FYI @seddonym - see here that fixing the graph increases the number of modules. I.e. we were missing many modules before.